### PR TITLE
Update wipe-mobile-device.md

### PR DESCRIPTION
**Pull request by msbemba**
Fixed typo

### CODE REVIEW

**Comment by msbemba** [comment]
@JohanFreelancer9 Please copy review

**Comment by msbemba** [comment]
Issue#https://github.com/MicrosoftDocs/microsoft-365-docs/issues/4417

File: microsoft-365/admin/basic-mobility-security/wipe-mobile-device.md [hunk 1]
```diff
@@ -76,7 +76,7 @@ Wipe a device for these reasons:
 
 The wipe is sent immediately to the mobile device and the device is marked as not compliant in Azure active directory. While all data is removed when a device is reset to factory defaults, the following table describes what content is removed for each device type when a device when you remove company data.
```
Comment (JohanFreelancer9) [inline microsoft-365/admin/basic-mobility-security/wipe-mobile-device.md:77]:
```suggestion
The wipe is sent immediately to the mobile device and the device is marked as not compliant in Azure Active Directory. While all data is removed when a device is reset to factory defaults, the following table describes what content is removed for each device type when you remove company data.
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review - some additional typo corrections, please.

**Pull request merged**